.ci/update-gherkin-specs.yml

```diff
@@ -0,0 +1,108 @@
+name: update-shared-specs
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-agent-dotnet
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+sources:
+  sha:
+    kind: file
+    spec:
+      file: 'https://github.com/elastic/apm/commit/main.patch'
+      matchpattern: "^From\\s([0-9a-f]{40})\\s"
+    transformers:
+      - findsubmatch:
+          pattern: "[0-9a-f]{40}"
+
+  api_key.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/api_key.feature
+  azure_app_service_metadata.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/azure_app_service_metadata.feature
+  azure_functions_metadata.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/azure_functions_metadata.feature
+  otel_bridge.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/otel_bridge.feature
+  outcome.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/outcome.feature
+  user_agent.feature:
+    kind: file
+    spec:
+      file: https://raw.githubusercontent.com/elastic/apm/main/tests/agents/gherkin-specs/user_agent.feature
+
+actions:
+  pr:
+    kind: "github/pullrequest"
+    scmid: default
+    spec:
+      automerge: false
+      draft: false
+      labels:
+        - "automation"
+      description: |-
+        ### What
+        APM agent specs automatic sync
+        ### Why
+        *Changeset*
+        * https://github.com/elastic/apm/commit/{{ source "sha" }}
+
+targets:
+  api_key.feature:
+    scmid: default
+    sourceid: api_key.feature
+    kind: file
+    spec:
+      file: test/Elastic.Apm.Feature.Tests/Features/api_key.feature
+      forcecreate: true
```

Comment:
what's the reason for forcing it?

Reply:
I just thought, in case the file is somehow deleted. This will create it again.